.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: "Build ${{ matrix.platform }} in ${{ matrix.build_type }}"
     strategy:
       matrix:
-        platform: [windows, ubuntu, macos]
+        platform: [windows, ubuntu]
```

Comment:
Any chance we can keep MacOS once merged?

Keep it in the matrix but add an if statement to the job to only run macos on main

Not sure what this would be in practice but something like

`if !macos or ( macos and branch == main )`

Reply:
We could follow with a PR to add this if it becomes an issue

Reply:
> Any chance we can keep MacOS once merged?

I was thinking that, but wasn't sure what people would think, @tomadamatkinson could you make a seperate PR with how you would do it, I think you probably know what you want better, this PR was more of a "here is a way to improve things"